### ISSUE TITLE
Make the lmms ui suitable for Qt changing (especially when setting the qt multiplier to less than 1)

### ISSUE THREAD

**Issue by YesImAsh** [LMMS/lmms#7440]
### Enhancement Summary

Make the lmms ui suitable for Qt changing (especially when setting the qt multiplier to less than 1)
Be able to run lmms at 0.5 qt without the video happening
it would be nice for people who have a TINY screen, or just dont want clutter in general

(i've searched everywhere on the internet, but there was no good answer)
(this may seem like a bug, but its more of a feature as there is really nothing wrong with lmms, i just want to decrease ui size)

### Implementation Details / Mockup

running the QT_SCALE_FACTOR=0.5 Will make a quarter of the ui visible; Find a way to make the lmms logo fit as a practice

basically, the qt is set to 0.5 and the reason why it looks like this is because it cant fit within the qt value

![Untitled_ - LMMS 1 3 0-alpha 1 668+gce17c9563 5_08_2024 1_47_08 AM](https://github.com/user-attachments/assets/b57917c1-3171-4311-be57-3a1ee87b6b49)
![splash](https://github.com/user-attachments/assets/4e00e589-fcc6-463c-b1bc-76965009397f)

### Please search the issue tracker for existing feature requests before submitting your own.

- [X] I have searched all existing issues and confirmed that this is not a duplicate.

**Comment by michaelgregorius** [comment]
On my system the splash screen shows in full when using `QT_SCALE_FACTOR=0.5`:

[LMMS_at_50_percent.webm](https://github.com/user-attachments/assets/5a11c4ac-6195-4347-8b1f-49737195a9b8)

There are some other artifacts but in general scaling works on my system. I think it will be hard to tell if the artifacts are caused by LMMS or by Qt when it needs to deal with scaling factors less than 1.

**Comment by YesImAsh** [comment]
Thats bc you're on linux

**Comment by michaelgregorius** [comment]
This is on Linux. You did not write what system you are on in the issue. :wink:

**Comment by enp2s0** [comment]
I assume this is probably Windows based on the titlebar design.

I will say that I don't think I've ever considered what happens when LMMS is run at scale factors _lower_ than 1. Usually the discussion is around factors greater than 1 for HiDPI and other large/high-resolution scenarios, and even then LMMS isn't great at it yet.
